ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -900,6 +901,7 @@ export default class ConfirmTransactionBase extends Component {
           isApprovalOrRejection={isApprovalOrRejection}
           assetStandard={assetStandard}
           txData={txData}
+          isBuyableChain={isBuyableChain}
```

Comment:
Yes, but `isBuyableChain` is not called, it is a boolean coming from the `useRamps` hooks, in this case the hook is used a HOC because this a class component https://github.com/MetaMask/metamask-extension/blob/2d1e883ddf01239cc51bdd88aef0ff9c8d814981/ui/pages/confirm-transaction-base/confirm-transaction-base.container.js#L318